crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -225,6 +239,26 @@ int crypto_sign_signature_internal(ml_dsa_params *params,
   /* FIPS 204: line 33 Write signature */
   pack_sig(params, sig, sig, &z, &h);
   *siglen = params->bytes;
+
+  /* FIPS 204. Section 3.6.3 Destruction of intermediate values. */
+  OPENSSL_cleanse(seedbuf, sizeof(seedbuf));
+  OPENSSL_cleanse(&rho, sizeof(rho));
+  OPENSSL_cleanse(&tr, sizeof(tr));
+  OPENSSL_cleanse(&key, sizeof(key));
+  OPENSSL_cleanse(&mu, sizeof(mu));
+  OPENSSL_cleanse(&rhoprime, sizeof(rhoprime));
```

Comment:
See comment above on pointer values.

Reply:
Thank you! I agree with you on signs, cleansed in [015ccd6](https://github.com/aws/aws-lc/pull/2017/commits/015ccd6f9adc8f8646d0df24945be1cec05c11b7). To be consistant wit ML-KEM I'll leave the cleansing of the pointers in to see what other reviewers think.

Reply:
Gerardo talks about these
```uint8_t *rho, *tr, *key, *mu, *rhoprime;```

Reply:
Thank you! Just wanted your perspective before making the change! fixed in [a8777cd](https://github.com/aws/aws-lc/pull/2017/commits/a8777cd91adbc6ddacbc0de027fce11eb8d3b4e7)